README.md

```diff
@@ -94,28 +94,23 @@ where you replace UDID and PackageID with their values you found before.
 
 
 
-1. Download [https://GitHub.com/jkcoxson/DiOS-instructions](https://GitHub.com/jkcoxson/DiOS-instructions)
+1. **Make sure you have a github account logged in beforehand** and Download the latest one that says *master* on the right and make sure it is MacOS and Extract it in Downloads folder. 
 
-2. Move the two idevice files to 
+	[Download](https://github.com/libimobiledevice/libimobiledevice/actions)
 
-`` /usr/local/bin ``
+2. In terminal enter 
 
-3. Move the dylib to 
-`` /usr/local/lib ``
+`` chmod +x ~/Downloads/libimobiledevice-latest_macOS/libimobiledevice.tar ``
 
-4. In terminal enter 
+3. Make sure your device is plugged in and that it is Trusted. Then go in Finder and click your device and Enable WiFi sync.
 
-`` chmod +x /usr/local/bin/idev* ``
-
-5. Make sure your device is plugged in and that it is Trusted. Then go in Finder and click your device and Enable WiFi sync.
-
-6. Type in and enter 
+4. Type in and enter 
 
 `` idevice_id ``
```

Comment:
The bins either need to be moved to usr lib or a cd and ./ needs to be added

Reply:
So I could just make it Chmod the bins file in the tar file and work out fine, Does that sound correct?

Reply:
Like cd to the directory with the binaries (like idevice_id etc) and chmod +x *

Reply:
So something in the lines of.

cd ~/Downloads/libimobiledevice-latest_macOS/usr/local/bin/
chmod +x /idev*

Reply:
If that's how the folder is structured, then yes